docs/intro/quickstart/ai/fastapi.rst

```diff
@@ -6,12 +6,16 @@ Using the built-in RAG
 
 .. edb:split-section::
 
-    In this section we'll use |Gel|'s built-in vector search and
-    retrieval-augmented generation capabilities to decorate our flashcard app
-    with a couple AI features. We're going to create a ``/fetch_similar``
-    endpoint that's going to look up flashcards similar to a text search query,
-    as well as a ``/fetch_rag`` endpoint that's going to enable us to talk to
-    an LLM about the content of our flashcard deck.
+    In this section we'll learn about |Gel's| built-in vector search and
+    retrieval-augmented generation capabilities. We'll be continuing from where
+    we left off in the :ref:`main quickstart <ref_quickstart>`. Feel free to browse the
+    complete flascards app code in this `repo
+    <https://github.com/edgedb/quickstart-fastapi>`_.
```

Comment:
Nit: We might as well use the `geldata` github org in all of the docs, but this isn't a blocker.